### PR TITLE
Feat(web-twig): Introduce Full Floating UI demo to Tooltip, update internal implementation #DS-1084

### PR DESCRIPTION
**Pull request by crishpeen**
Switch to data attribute placement instead of modification class based.

### Issue reference

DS-1084

---

### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md).
- [ ] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-twig/src/Resources/components/Tooltip/stories/TooltipPlacements.twig [hunk 1]
```diff
@@ -2,30 +2,30 @@
     <Grid cols="3" UNSAFE_className="mx-auto" UNSAFE_style="align-items: center; justify-items: center; max-width: 30rem;">
         <GridItem columnStart="2" rowStart="1">
 
-            <Radio name="placement" isLabelHidden value="topLeft" id="placement_top_left" label="top-left"/>
-            <Radio name="placement" isLabelHidden value="top" id="placement_top" label="top"/>
-            <Radio name="placement" isLabelHidden value="topRight" id="placement_top_right" label="top-right"/>
+            <Radio name="placement" isLabelHidden value="top-start" id="placement_top_start" label="top-left"/>
+            <Radio name="placement" isLabelHidden value="top" id="placement_start" label="top"/>
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/Tooltip/stories/TooltipPlacements.twig:6]:
```suggestion
            <Radio name="placement" isLabelHidden value="top" id="placement_top" label="top"/>
```

Please also check the other Radios.

**Review by adamkudrna — APPROVED** [review]

**Review by adamkudrna — COMMENTED** [review]
Just one thing: is there a chance to improve the `README` according to the `web` package? I'm sure our users will appreciate if there are instructions for placement, interaction etc. 🙏🏻

**Comment by crishpeen** [comment]
> Just one thing: is there a chance to improve the `README` according to the `web` package? I'm sure our users will appreciate if there are instructions for placement, interaction etc. 🙏🏻

TBH I don't want to do this now. as this PR is mainly to match demos. 

We are already working on including the floating UI directly in the Tooltip and part of it should be an overhaul of the Docs. I don't see the added value to refactoring docs now.

**Pull request merged**